src/components/appRouter/routes.js

```diff
@@ -0,0 +1,4 @@
+export const routes = [
+	// Add your routes here
+	// { path: '/', title: 'Home', component: () => <div>Home Page</div> }
```

Comment:
Don't you think it would be better to keep it this way:

component: YourComponent

And remove the upper comment as the filename itself is obvious. Very minor changes but since it's a template, I thought maybe we keep it very straightforward.

Reply:
Updated